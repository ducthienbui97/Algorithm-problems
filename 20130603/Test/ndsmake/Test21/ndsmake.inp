2052
-931080697 -238798161 -122136028 -781798491 209272106 967725226 -600029526 874484698 278825124 -463004185 -608137662 -670985974 -743304943 594239135 770793725 925488908 371914124 -285727874 882263527 -876562080 147378542 -190414528 -493339750 611368289 -94085147 843832582 476087678 -640105584 -377099180 262700852 -997357310 -690431974 -844125340 -492949587 -348334876 -317935860 202688133 197847928 -346041490 -170464723 651897919 659085326 151897754 778387865 222636484 -182728733 -220777313 -559534410 164874437 294749994 -557956044 -810315156 986716186 803103423 -719607568 390783164 -580179292 -618047872 313648671 -989021074 -623078063 -349333346 -163655437 578695212 -437265715 669821150 892342503 -239389764 -936444519 -843727028 -971632074 671408465 -918050997 683403915 -470592281 -581756014 547257671 -670149028 534224586 -502187153 323856622 134872217 227673657 190010 -450913486 -307843761 32315328 -952715534 814492269 414164423 -107744242 764635721 -258126178 -199031648 402529029 341057489 -628386320 13116609 161224861 -471744871 -274443564 -523324695 427343559 107755851 -221732129 672203312 -552181479 -840498618 -205172516 -172613197 266456458 -917436948 507972522 416067292 -526458846 696033116 553848256 -866054174 -123615084 748329302 19331239 -281171784 -268310028 523712201 -393740924 811530929 729322191 -403799270 -775714666 181106395 -180721286 -353821421 -26746149 326097910 246549379 447383147 -109442094 -929482224 78115401 -150953490 159395164 -723692098 986893992 267692617 487336892 -199718028 431485885 -930234478 -761505497 -602920772 -138589965 -631227053 795130220 233633305 536058804 -346458852 958989847 921002394 21341495 630393130 139980933 -14243797 -587303840 -617353706 214126648 203587477 -154776492 -568706256 880790937 13124496 -154001139 -629895851 396035524 -395202869 -390190792 970194115 53731565 -476388855 732525484 -564026003 -224887963 125508230 -340369372 156561773 732471222 585715173 -630823635 -815269875 -991686593 628274920 133785586 251774299 -32956834 -278422563 -170520182 981617712 23338235 -804558280 -8134019 497351326 -963262987 492289879 980142652 338587438 -374039797 859893626 551780422 -885791485 582484356 564799069 -355430942 -636472731 210799212 -397872896 -759532931 -182021991 -196571375 766124496 -335044012 -945465208 586729914 -452777970 -343969853 -849586150 -410941866 -16994169 -513455336 -752536862 625403853 540285864 -121814839 923749489 456728038 818492786 -174291857 891398311 -642256686 59381786 457033323 451928023 121588866 963002445 34083076 -215769381 530248794 714393748 -259515102 -784542143 -721438510 -798492506 91253410 -385925235 473795563 -954959772 502814739 844461200 -409850497 184389566 -802858315 -876966211 -914173484 299255774 -684634520 6730102 647768530 -745885553 38967196 -531363674 876647719 -100399289 -708951346 -724822901 -989406089 855231475 10183085 -566895075 -158996777 38316939 -810314260 970933549 -643722075 273229603 400575635 -125582828 -245775604 -182473685 -945191428 909557250 -447825612 145044097 55331719 886575358 -698570253 -165453121 -281373406 -573297682 703300883 -259684672 -558172135 899202614 116346495 373380306 680186424 -219297489 163631422 274209546 -796903713 103247244 752326150 -804754255 983700679 -250953340 -569011285 -849981272 -692517092 39370465 369476345 945436642 -885073843 433493022 516781593 -291302919 -625890348 -483189346 694300092 966359969 790928534 832247680 895921493 873768079 -131990099 -608111602 695010352 -10099437 -392781286 94118159 -64058410 502167274 521803651 203388870 -148469236 -457614491 917029335 -551050766 113455370 710043991 757997330 350736964 -665964714 977526917 539995110 -13106915 579577227 -300017198 627101681 143453881 138605081 -419160556 -963128501 -722767633 -384054744 138333467 -930263408 554059358 224696717 66007765 695835194 -633386601 11287320 441219784 -283883264 861103479 873638777 309746107 6991305 -276284264 -881250443 -171014325 393188830 415600847 -512540003 232098629 105103404 369938317 -820078400 720644949 -157204679 190058730 -579269519 -985035402 336933757 -356288869 167535499 358109722 -641921211 313849041 569526162 507994592 472599270 -133040246 -83090888 981441778 -955389426 93966964 -458948613 -273750530 -925973450 357500281 -502023445 615686707 -85983253 257281930 798190727 637231852 641772622 -633381428 107825531 -43878106 -680218948 962787389 -958698837 -385600389 475979769 486278877 849962858 -169267436 894469751 -810513068 -798745465 -341789770 921399864 -937948622 416099539 -829138166 207518523 291845735 167487782 -897500855 598396506 989547861 -307484471 633219552 207748719 -602643966 891105056 -41351103 197439065 660966905 352778884 401763139 -96823013 165737405 15216175 -884380361 925350434 -679792917 29747536 -482891128 224163751 -480678933 378862911 106316046 -719662805 887821515 109525487 252228639 -266009384 -227426642 -546609102 -175654746 -554437135 -773019534 -591448054 91331724 -359990821 -48934453 285790260 889276590 296135553 -196129572 -293531832 -75868142 -370727335 -492781187 -740822173 111379782 -536574216 140762577 920271622 -275667082 -344077080 -512402695 -583807528 -486623924 -378353135 888776745 215550642 -299817546 -487078717 99792163 318900406 672888050 -20163339 -979639921 35885447 375446089 -820096703 -12975460 -399044918 -515428666 -323796702 292056820 -474684613 831303283 132104152 -66651272 -885546544 380788993 -903149804 -55091332 -345337466 -293696925 857005943 180988772 -494894444 -289531615 -461662365 -108281128 633257055 850811150 -829606882 279865810 794179825 -211981501 63415659 947997728 -899521400 -958426003 704504882 -322304073 -201764179 -896404059 -212729673 849342491 945096800 -43992634 459201428 -523578198 562849186 103632321 49217547 341549442 -596540258 104698234 12444944 -884771939 -362364656 486235628 -618142666 220775057 378003123 -341751861 297764632 133272940 649475300 -407217996 -269767146 -449467075 400748387 -252120581 -964683488 884574123 -379610419 427920992 677493796 -327481810 -544910259 -26436292 -866965429 -647177683 634776310 -160649044 -205532439 162647412 -540110329 666779624 -893061566 422232267 370386044 -477594512 -791168460 788778945 667121620 352351836 680090816 569243521 64927777 766755050 781960206 591938717 -62252829 -5989839 835280225 310112569 -446509348 228591847 758380637 -700414992 -199154625 -675735588 673966430 -136506455 -756742243 899725173 -826601089 202467229 647870521 -668700672 893454436 -524482374 -448680672 -694003227 437437564 616377638 -319069589 -453260328 -157124949 -755097433 507919797 528697893 44769123 -856920123 -671741314 -428601407 -247807283 -359646298 -873802431 -452724953 -159406408 129894376 -211028316 -487638227 734345165 -45183193 -823942108 876450916 147642820 -915326680 900597832 -601065019 426653077 576143233 -226849490 946610041 -677482052 188762582 -571673869 29391109 193845310 -478530961 -289147456 478406189 659980066 177527109 397254650 336639326 -908106491 627600744 665143620 725392479 802459503 848298476 -676906466 -332337077 460946498 -886069337 -947641559 -816580532 390861902 728656259 -956350167 -667120389 -252115622 -150148137 841729917 700634928 -350150314 -914751631 -504246574 -953730772 781388026 -855160315 47457130 948992423 -969876069 886978706 39290072 -289734145 881240181 -217156124 226223959 -666333811 -640439141 -103413966 125972678 506608660 172404088 233001191 -14251196 634132819 736351233 -459339932 761809850 -334791559 565387398 -173021171 102317113 411293408 -873484347 -761097743 -327866061 -203693063 67849337 -461260632 798816503 548734650 -766704792 -81974029 279483331 783197435 964690787 634927618 234023404 -832980561 837547453 -332973012 -889900792 -350131415 -824147518 273258643 145144026 975081247 423799701 -390441645 964658334 -498571076 539374298 834285095 -975040621 -394195146 237295256 -581792160 707174143 269642745 -786922394 217330061 -868850028 918864995 -756475738 634840262 204558573 -244194023 -534728750 -758656363 -919564745 -665229052 436894446 54551426 922233581 -938986004 -47372003 385244474 218894523 409790639 523668059 658568068 -380335331 517185305 384160990 564644991 363236723 436986270 -744923870 230335868 854367603 588956584 343193489 924554821 10655004 856074715 386276081 790820383 -199992565 236772352 164292937 -972561552 -811763074 604216685 -223774560 768874728 -643512535 -774341563 -619979161 271462319 421748742 -403977573 452455110 595703544 842359484 -481085907 748440958 541678729 -318492876 733775315 -408139992 272688860 526181687 636204529 494894312 -610044074 566460345 194646773 548998607 -478773871 141900284 -909876273 15158742 671772131 841105812 899930715 250506406 -579983980 -605328228 -638258021 38453623 -628923192 -410851000 827293001 764936828 580999538 690699841 -908031445 -416977527 515778537 318242093 -151915898 256392884 -661338180 -858866406 -286012965 -597547375 -849336366 968196087 710354313 331595170 -860932991 -591421188 -802483656 -454860075 122646047 91954224 -226668067 157774255 990360805 838756768 648990571 86339860 75542767 -635006405 446061481 153691317 -593201057 -471107507 -235330273 -354743530 -965266014 -824840810 -878346966 638349883 25472970 -880619258 -999056337 245886191 127579931 89656856 -373737748 665293488 684103585 -339478584 -8923414 224334906 -158608386 -349365966 232458255 801001134 -114521534 599799241 -862286512 915515231 -71301901 -13905104 44428931 948967994 -740681172 138531154 -623147825 -405923103 612425847 378842955 269899776 909274625 -257451243 -453346748 -319535821 224194246 137946814 -552614759 -27309566 570553352 -813064913 780181526 -582564233 -978591551 -380237370 711885784 -923202710 -68722456 -529107939 -52081992 -861518916 833239309 127564211 596048714 397811265 -61887878 -322772569 90244490 -725796892 501957487 -515444169 -897188544 -507673762 -800761729 728054786 668985360 -920609905 -483764148 -266781144 -519880374 -948358383 -967884938 65313321 -581920052 -529312040 59302393 -610955839 658991523 917127539 -53343411 97152214 -511797922 192225723 -232784676 -861949292 814671234 -290590735 -59210216 -361612686 -294630488 -754353750 767392098 484557849 328273853 -603456565 878305556 939683377 -418189949 553130137 -396466049 11438101 684759476 731720256 -261472665 -594683322 -181340992 794348312 883459909 -761460906 396755980 -578589189 423593659 184554119 318794925 -181821775 -455060021 479403723 328361842 967964304 -96129886 -57920959 -283351066 874777843 -503224195 946093324 134899139 -690719924 -988084072 -274790249 -463964419 822700453 733933401 -9526679 564217236 922219406 -326083974 744561842 -207216391 -683194649 661456120 -100263468 862681179 360621687 -256831762 -762350534 -783539445 -629504111 -140066069 -481875532 -999937950 151654271 -409420657 -932466520 -808714022 -734865707 -790563304 -217057963 -640935883 877221498 501617067 -870485802 -919738075 43649340 600579381 -620015194 695475870 265454893 555513697 155295343 586355517 -574163001 800732698 -5082112 -103834282 -495451607 -618906978 -543428514 831182720 840314172 873727583 392273793 -996984150 619465183 -798293609 -828358894 716770565 740945014 -745743544 508152072 -107537819 -640339261 723500738 -706414325 695075612 -550347200 -789162907 73161100 696008205 -140255717 -847655518 -904557073 860729369 152124995 499078643 437736006 -439660265 -974669997 -584413071 -17783975 -802936307 246969002 83113161 87161245 199364966 924127837 -422000957 438409848 504764711 89306073 77953607 669332398 -553541140 239362676 127663398 621002071 -316543975 -892149051 -922028372 615351049 -689349735 -431954625 -542552981 -123306720 -325161144 260604991 -156346742 460573185 -356440145 -800669472 -321381537 -45820030 371307238 -776847107 -234330248 -267655180 892281727 -448840263 -519421509 920357147 -459991320 -561693921 -564770559 355009425 -93457892 -389247719 817728674 -306776196 -671250457 669758950 -72188986 657893304 -766853351 483932835 106627047 -506719419 -539074753 907822775 607371851 186919271 -53855554 -975762673 -534121751 -32642657 772912897 523075495 -495130072 732197447 791057776 793012421 -932236569 384318602 -703687941 882518709 -73455587 -154411942 929485988 -820147702 509348379 531947327 -716786511 -8874921 -403616087 -416526243 230297008 -993521793 239202102 358084547 -146652933 658246413 431302697 -800893872 -215358626 -735449807 -754541739 -108461502 -335851057 520623448 880338828 732045404 -67973529 529065802 -911993421 -560532190 274459283 -156178286 -316802538 955728249 419562207 -80850985 -155426024 177924909 249689139 -567199728 -705120324 86603751 -819669994 -226180626 407170219 364263176 -927472598 249457524 -584420486 -757972353 523874039 -853138063 531396888 -552033156 -421981033 255145617 322020584 522791624 -775460598 453945304 511192307 -249539632 -255989023 -891534129 -418000102 844744496 -239094375 -66084868 396665699 692529825 -971575404 -183941614 673439831 -41217627 957269672 -330118282 -181321236 -582702650 167840047 824292188 347747865 239612033 419428454 -629530691 831686863 42553371 -588799942 777662014 -254065686 26578283 -166644271 761546752 294586938 495146484 -523640434 566440042 -963523658 91903745 -698834437 275588832 425707547 -643246508 223650199 -485498081 -518527746 315652545 -332031411 456885761 -344829898 -468842691 -325024868 -219313667 200883559 394531996 752137264 -634360394 -421809620 -264130188 -203734243 74878241 968426167 980162327 614171145 190593762 844042000 531152330 -578345886 500454945 236685839 572243793 -696501372 431319248 927711017 -500962018 236919088 -365128209 891979967 -331655201 -136000197 -7769836 -489490967 -214208489 -295742306 -319772439 -498794799 513297401 -905090628 -926315081 788540954 904343212 173748806 469309833 146751240 469814770 561440887 -568641933 776147859 -217921390 -834427585 -848982094 -524609417 910323203 -992796756 -154658206 -336684952 -490989901 -10960796 -926208042 292737123 183727860 451095375 495891351 -723208416 -304744828 -812023471 -791978244 737588919 596360245 -982586269 -969912590 739021123 -176576859 486374409 -955013919 966064552 409604694 888024204 333858825 58936122 -509342144 466872535 596815308 313917195 -933769579 -775333125 -992830587 -417625050 301036179 -711021036 -536219824 924777722 -478610201 732039331 -270228071 2904445 -245076558 -501795431 -372638206 136771442 16116731 -648178407 -730010206 -120425523 -251458014 878100525 396551025 210771208 -676149767 901806325 -306941164 -491066096 511521046 918351851 -107926149 -747844682 628659057 524893519 785022161 53115152 -106764160 374859879 -709225088 -122676095 -392186413 -713365943 -758906566 423987960 -472562015 -211186528 -848518041 463711326 -255676085 -834743423 196590125 577167374 -639309376 -840294534 -286760763 366846805 -315400179 -67063327 575918394 273278892 427466664 -223873190 -209348416 -607364607 -919250599 429849718 -815280722 -23884495 -270802490 268388982 -829466087 -732554861 -774320487 -481325523 -599923794 458357840 -528053501 131992429 425450506 -513808736 130866525 77507108 952229255 -941283974 -718017168 121361531 -616808211 -838797407 716535892 -492693561 -499741450 703317718 -729702169 51820771 -686220633 -981798851 453724850 530000595 -397416356 -534054291 55230751 -321883950 -970589116 -718741537 788682393 432435760 269156465 766535087 36523667 615507604 542515191 641032085 -779222803 -707943921 460983271 -240563596 702491064 204301056 -529753384 439346317 35101485 996052555 715529387 -921755971 -531564081 32049385 -821460193 767402845 288384658 819624090 565603164 -412208083 217881425 491929585 581979531 -220252920 653712720 -62486825 266078065 106794147 937005516 -477448993 13911479 348540598 -333129945 947052847 720640126 -778689861 -730833380 -534446876 -888791995 -317955059 -395231397 7126182 -541622533 11452536 -183700221 -399612449 835553327 998643592 711363133 566476015 434894879 509371336 509078732 -889015097 -577558286 915150623 -230144128 398098528 550945254 385264729 -317503045 -111387271 730553270 -765129991 721531363 768400049 38621150 893755435 -263315442 -930631791 -343611990 22299459 -489736194 -448859602 -257158064 631680086 960277991 618322129 -149092689 -584172435 -568739578 -926884962 746231896 672064832 421373269 270036126 469623945 -367319842 -422522260 -926135013 -760133433 -171441427 650457354 606000259 -755508992 566073628 685879771 -497854900 65242425 12201132 730819407 313155322 882994846 -760920678 593332210 -537107766 824331200 488157035 -667595567 826201306 -863983469 876840456 277507656 -958074736 -236723052 523957810 287481849 -45912149 -71716605 62671779 -480710183 628446948 901378382 -742160701 -600265628 129540928 239402894 -401386493 596934129 513556981 225081130 293527258 779741984 -163226378 -479803495 487823962 154014831 86550045 -540544698 751352174 -513801492 -998970636 30273164 -658788174 -896676347 683072289 -803116320 -2311185 -270726021 -879588501 -807141070 486285941 879615853 507748764 -484996217 975773361 405775318 -682362667 -362167463 -299618373 -338867621 -224582940 151101496 326767662 -48557764 474906572 626015337 795852389 402291913 -50597605 459569762 -34144346 539154644 -354824750 -601071482 -883671073 817125034 -190139310 644802860 480054757 -930167508 -267695757 -347174584 -788516395 618492046 371537848 -851885157 375325612 -144778868 529358637 258992017 -381426428 -172116513 -50871116 308474687 7022932 -619455092 -270201632 -448668607 183358929 -677552349 173681046 236647238 401198081 -543625804 -200172939 225119540 335108406 311399362 568330719 -901555095 -614857817 -233348265 -43945815 -775660515 722784044 -449850063 -286487973 779669475 -601631851 -749754086 -762709682 -248209752 -442404771 183911856 803503030 -854109388 -964646728 -66726402 281762521 369870535 710600965 206332623 -268717620 703201199 786795680 -776069501 896094311 -710608773 -410074443 827775940 -392119144 977828532 -80244430 -454146681 30841768 460178051 -630191375 -907419364 160248971 -820040090 271432566 547821640 -872330461 561748160 -393651146 952292299 487650082 964323750 418581451 -663672890 -325638979 -729259664 -154352878 -275438465 -775861774 -476273906 234267244 -382059552 -241026355 662360426 565411962 -996662613 407112627 886464746 180710007 561926858 -652462518 -968737007 -261386189 -51633980 539567880 975704645 281589160 -726174132 -970532521 -734760021 810607675 -739023884 -885470110 550417808 510188543 -960349569 -220174945 -486788598 -620296109 -578151111 999728375 254860510 -657076851 -816371720 807795383 166078418 -422826705 -611810481 270000240 23931032 -703244044 863154914 -869543329 918817217 854223173 -703287310 -343697360 698831200 794020036 296739463 -504621085 660780860 -603683068 -577879239 499987660 237429937 -952966052 191664026 993090740 -412805085 183797969 -998406882 -125352416 -379979728 931555194 -84116828 -102043119 728495421 551635837 225215615 166631581 167581550 -814973403 -389732965 -885429424 736672730 437627148 -415199311 -386490393 107059341 -964930674 -963346750 809070618 961526244 -102618944 -966173982 -633155645 775166548 873392690 821091387 -639928570 -549237690 -161443251 454748038 -570919597 -756311413 -117852774 859668911 671747960 -754755813 242902176 -689355002 927806868 427193899 461841076 623217585 838275870 -357537077 -289607292 12301690 -606316583 661285714 611298196 609587372 -261614559 35657483 760982231 941288819 -748664393 -360515740 -312146841 -919158704 340586729 39196307 857758473 -808710822 130762177 628284070 613184236 -321125074 -197504088 705430913 336317209 696305267 -556181211 924220018 578886665 -513905144 -987142718 -284523662 893644112 375177165 -690506737 -614338565 126063736 -833203701 739008960 14878592 -125694231 151963009 181487601 -343021170 133515523 108975499 713135485 -957053874 299429310 -771263534 -155005494 367258798 -674768075 -946758305 727880460 479859754 -822133897 -909950159 -643054953 536262061 -222502486 622298502 276362731 819483727 424551824 894729619 -476529451 692788840 809473203 166121506 531322874 -995467561 -228397915 407809774 318631259 30638238 -340426194 -469191446 30592978 915450269 -780403414 -368083781 623015556 246354495 948076146 -121531984 -83458421 -955308306 455562586 444472428 -702288247 384411073 721897242 959629009 754946018 150257889 -705009928 -189366012 939112596 169818747 -973969333 526347190 -562942635 519310801 518506282 389932158 677673801 -670148756 552112566 -881281438 -837457059 932073068 -427934719 298094709 -908193469 -830783233 88901786 -634321766 707694602 -292837904 -828231753 698839142 -349548968 436958551 139759357 -375631324 -782175161 -374728081 -699773970 196717554 230894810 -369332092 -550669585 -86952184 -341326565 93609227 -780387083 -112244455 874865362 735492300 -231193140 80965074 -271667997 424761914 -697918204 -459155841 -593338555 -144821533 -384844936 896590285 -10591077 192778754 655689596 -87548502 486322033 706788914 295842210 -388348600 -325964524 767506493 836457513 -946217663 917752846 -584836845 558718723 -790614059 998977596 268836054 154466946 -235369650 864308878 -833352430 -544691554 417186948 26659419 779459813 85335933 625053279 175235885 -833934593 192536620 721206098 546100904 961002461 -713890470 63809061 459677717 -587575938 -611856478 -378912805 -30117295 -929656482 -831826282 -678108926 972392555 525215099 829459968 -626060594 -225238601 -682689213 -426668080 16411907 -499620959 44927711 -652126632 -234244895 546054746 85548562 -176339356 -480035037
