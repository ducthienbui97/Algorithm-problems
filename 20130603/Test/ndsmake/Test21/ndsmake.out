1031334183571
